python/ray/tune/config_parser.py

```diff
@@ -103,6 +103,20 @@ def make_parser(parser_creator=None, **kwargs):
         action="store_true",
         help="Whether to checkpoint at the end of the experiment. "
         "Default is False.")
+    parser.add_argument(
+        "--keep-best-checkpoint",
+        default=0,
+        type=int,
+        help="Number of highest reward checkpoints to keep. Value of 0 (default)"
+             "disables best checkpointing."
+    )
+    parser.add_argument(
+        "--keep-checkpoint",
+        default=0,
```

Comment:
Default od 0 is doing exactly oposite than the one for the `--keep-best-checkpoint`, and it is counterintuitive (0 should not hold any checkpoints)... Default both to None.